packages/assets-controllers/CHANGELOG.md

```diff
@@ -5,8 +5,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [19.0.0]
 ### Changed
+- **BREAKING:** Bump dependency and peer dependency on `@metamask/network-controller` to ^16.0.0
+- **BREAKING:** Add optional `networkClientId` and `userAddress` args to remaining `NftController` public methods ([#2006](https://github.com/MetaMask/core/pull/2006))
```

Comment:
Adding optional properties seems like it should be non-breaking :thinking:

Reply:
Oh I see, this change is non-breaking but this is a description of a wider set of changes.

Maybe this description just needs improvement then. Perhaps something like

```suggestion
- **BREAKING:** Parameterize remaining `NftController` public methods with user address and network client ID ([#2006](https://github.com/MetaMask/core/pull/2006))
```

cc @adonesky1

Reply:
I saw this as an overarching description that includes the bullet points, but on second thought I agree that the breaking alerts in the bullet points do the job, and we want to be specific about where to put those alerts. 

```suggestion
- Add optional `networkClientId` and `userAddress` args to remaining `NftController` public methods ([#2006](https://github.com/MetaMask/core/pull/2006))
```

Reply:
Ah OK, this works too!